media-sound/naspro/naspro-0.2.91.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Architecture for Sound PROcessing"
+DESCRIPTION="NASPRO core is the portable runtime library at the bottom of \
+the NASPRO Sound PROcessing Architecture"
+HOMEPAGE="http://naspro.sourceforge.net/"
+COPYRIGHT="2007-2011, NASPRO Development Team"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://sf.net/projects/naspro/files/naspro/$portVersion/naspro-core-$portVersion.tar.bz2"
+CHECKSUM_SHA256="00d61a9a18aacb8d0fc3e21ae56f005484111a52bdc47ce64677d568908bfe2f"
+SOURCE_DIR="naspro-core-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
```

Comment:
Thanks for the pointer to the 0.2.90 patchset, missed that one, on the note of the 0.2.91 patchset, haikuporter seems to build ok without it, in Terminal I need to aply it  (is gcc stricter in Terminal then with haikuporter?) ...

Reply:
PS, for as far as I could check there is no test for 0.5.1 ...

Reply:
Good catch. The patchset is not required. (I guess it would have been required for compiling with gcc2.)
I don't think gcc would be stricter in Terminal than with haikuporter, but I might be missing something. (Maybe hp passes some environment variables.)

Yep, regarding naspro 0.5.1, I also noticed there weren't any tests...